CLI/Testing/Cxx/RegressionTests.cmake

```diff
@@ -0,0 +1,228 @@
+#-----------------------------------------------------------------------------
+# Regression Tests QINProstate001
+#-----------------------------------------------------------------------------
+set(inputDataBaseName ${CMAKE_CURRENT_SOURCE_DIR}/../../../Data/RegressionTests/QINProstate001/Input/QINProstate001)
```

Comment:
Here `CMAKE_CURRENT_SOURCE_DIR` can probably be replaced with `CMAKE_SOURCE_DIR` that I think should point to the top-level source location. This would be cleaner.